### PR TITLE
Display network name for IP in shared networks

### PR DESCRIPTION
**Pull request by ravening**
## Description
In the list publicipaddress api call ,display the network
name if ip is associated to shared network. The network
name is displayed as of now only if Ip is associated with
isolated network

## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

## How Has This Been Tested?
Before the patch

```
(local) 🐵 > list publicipaddresses ipaddress=10.1.1.1 listall=true forvirtualnetwork=false
{
  "count": 1,
  "publicipaddress": [
    {
      "domain": "ROOT",
      "fordisplay": true,
      "forvirtualnetwork": false,
      "id": "eb38235f-e478-476a-bccd-bd4c95d92c8c",
      "ipaddress": "10.1.1.1",
      "isportable": false,
      "issourcenat": false,
      "isstaticnat": false,
      "issystem": false,
      "networkid": "791432-089-4bac-8a12-0c1efb",
      "state": "Allocated",
    }
  ]
}
```

After the patch

```
(local) 🐵 > list publicipaddresses ipaddress=10.1.1.1 listall=true forvirtualnetwork=false
{
  "count": 1,
  "publicipaddress": [
    {
      "domain": "ROOT",
      "fordisplay": true,
      "forvirtualnetwork": false,
      "id": "eb38235f-e478-476a-bccd-bd4c95d92c8c",
      "ipaddress": "10.1.1.1",
      "isportable": false,
      "issourcenat": false,
      "isstaticnat": false,
      "issystem": false,
      "networkid": "7932-089-4bac-8a12-0c1eb",
      "networkname": "shared-network",            <<<<<<<<<<<<<<<<<<
      "state": "Allocated",
    }
  ]
}

<!-- Please read the [CONTRIBUTING](https://github.com/apache/cloudstack/blob/master/CONTRIBUTING.md) document -->

### CODE REVIEW

File: api/src/main/java/org/apache/cloudstack/api/response/IPAddressResponse.java [hunk 1]
```diff
@@ -150,6 +150,10 @@
     @Param(description = "is public ip for display to the regular user", since = "4.4", authorized = {RoleType.Admin})
     private Boolean forDisplay;
 
+    @SerializedName("networkname")
```
Comment (rhtyd) [inline api/src/main/java/org/apache/cloudstack/api/response/IPAddressResponse.java:153]: @ravening can you change that to use `ApiConstants`?

Comment (ravening) [reply api/src/main/java/org/apache/cloudstack/api/response/IPAddressResponse.java]: @rhtyd Done

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
Strictly presentational change, full smoketests are not necessary; packaging and Travis/smoketests have passed.

**Pull request merged**